packages/plugins/jwt/src/plugin.ts

```diff
@@ -0,0 +1,242 @@
+import { createGraphQLError, type Plugin, type YogaLogger } from 'graphql-yoga';
+import jsonwebtoken, { type Jwt, type JwtPayload, type VerifyOptions } from 'jsonwebtoken';
+import { JwksClient } from 'jwks-rsa';
+import { type OnRequestParseEventPayload } from 'packages/graphql-yoga/src/plugins/types.js';
+import { normalizeConfig, type JwksSource, type JwtPluginOptions } from './config.js';
+import '@whatwg-node/server-plugin-cookies';
```

Comment:
This is a plugin you should add.

Reply:
What do you mean? It should be added by the end-user. It's here only for type-safety of the plugin code.